### PR TITLE
completed task 2

### PR DESCRIPTION
**Pull request by kellysum**

### CODE REVIEW

File: db/app.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+const express = require("express");
+const {getAllTopics} = require("../controller/controller")
+const {handlePsqErrors, handleCustomErrors,handle404} = require("../error")
+
+const app = express();
+app.use(express.json())
```
Comment (ghost) [inline db/app.js:6]: We aren't posting any data yet, this isn't necessary at this point.

File: db/app.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+app.get('/api/topics', getAllTopics)
+
+
+
+
+app.use(handle404)
+
+app.use(handlePsqErrors)
```
Comment (ghost) [inline db/app.js]: I'm not sure why you have this at the moment as you're only testing for a bad path.

File: error.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+exports.handlePsqErrors = (err, req, res, next) =>{
+    if(err.code === "22P02" || err.code === "23502") {
```
Comment (ghost) [inline error.js]: Currently nothing your server does could cause these errors. You're just doing a select all from a table. Please don't add code outside the scope of the ticket.

**Review by ghost — APPROVED** [review]
Looking good, remember to only write code for the ticket you are on. You don't need all of those error handlers yet.

**Review by Hard-Luck — APPROVED** [review]

**Pull request merged**